cmd/nerdctl/container_run_windows_test.go

```diff
@@ -108,3 +108,17 @@ func TestRunProcessContainer(t *testing.T) {
 
 	assert.Assert(t, !strings.Contains(inspectOutput, "hyperv"))
 }
+
+func TestRunProcessContainerWithDevice(t *testing.T) {
```

Comment:
is it possible to add a comment here to trace what you said earlier :
> Obviously this isn't ideal given that Microsoft might decide to change the path in the future (breaking both this unit test and the one in containerd itself)

Reply:
Sure thing! I've now modified the commit that adds the unit test, so it includes a comment with the relevant details.

Reply:
Thanks